api/database/migrations/data/2024_04_01_222259_create_draft_lyrics_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::connection('data')->create('draft_lyrics', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->uuid('track_id')->unique();
+            $table->json('document')->nullable();
+            $table->timestamps();
```

Comment:
I do think we should add a unique constraint here on `track_id` if this is a one-to-one relation as we want.

Reply:
unique constraint is already added at database level
```
$table->uuid('track_id')->unique();
```

Is there something else you are looking for ?

Reply:
my bad, i missed it - too early in the morning!